### DISCUSSION TITLE
how does 'Exit Sleep Screen' shortcut works?

### DISCUSSION THREAD

**Discussion by bheeshmpita** [koreader/koreader#11779] [Q&A]
Normally when the device is in sleep mode, does the touch screen is still working but not registering the touch or the touch screen isnt active at all?
Does enabling the 'Exit Sleep Screen' shortcut means it is actively registering the touches and eventually leads to a depreciated battery life?

**Comment by mergen3107** [comment]
When set it up, you can unlock the ScreenSaver by this gesture.
I don't know about the impact on battery life. I use this feature as a multiswipe gesture (replacement for a pin lock), in Kindle Scribe which has magnet cover. So stray touches aren't an issue for me, and I didn't see any bad battery performance because of that.

**Comment by Frenzie** [comment]
The device goes to sleep the same way it always does. It's the waking up that's different: it doesn't change the screen until the gesture.

So if the device wakes up and you just leave it, it might take a while to go back to sleep, but it's otherwise identical.
